submodules/misc/misc.js

```diff
@@ -402,6 +402,66 @@ define(function(require) {
 						}
 					}
 				},
+				'set_alert_info[]': {
+					name: self.i18n.active().oldCallflows.set_ringtone,
+					icon: 'play',
+					category: self.i18n.active().oldCallflows.advanced_cat,
+					module: 'set_alert_info',
+					tip: self.i18n.active().oldCallflows.set_ringtone_tip,
+					data: {
+						alert_info: ''
+					},
+					rules: [
+						{
+							type: 'quantity',
+							maxSize: '1'
+						}
+					],
+					isUsable: 'true',
+					weight: 20,
+
+					caption: function(node, caption_map) {
+						return (node.getMetadata('alert_info') || '');
+					},
+
+					edit: function(node, callback) {
+						var popup_html = $(self.getTemplate({
+								name: 'set-ringtone',
+								data: {
+									data_cid: {
+										'alert_info': node.getMetadata('alert_info') || ''
```

Comment:
Since you don't pass any other data to your template, no need to have it nested inside an extra object, you can get ride of `data_cid` here.

Also, `alert_info` is a valid identifier, no need to quote it.
```suggestion
										alert_info: node.getMetadata('alert_info') || ''
```

(Don't forget to update the reference in your template from `data_cid.alert_info` to just `alert_info`)